internal/ui/update.go

```diff
@@ -383,15 +380,14 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				cmds = append(cmds, deleteActiveTaskLog(m.db))
 			}
 		case "s":
-			switch m.activeView {
-			case activeTaskListView:
+			if m.activeView == activeTaskListView {
 				if m.activeTasksList.FilterState() != list.Filtering {
 					if m.changesLocked {
-						message := "Changes locked momentarily"
+						message := changesLockedMsg
```

Comment:
All your messages, should be prefixed with msg, not suffixed.

It would help you with code completion